frontend/src/app/components/reward-stats/reward-stats.component.ts

```diff
@@ -24,10 +24,11 @@ export class RewardStatsComponent implements OnInit {
           return this.apiService.getRewardStats$()
             .pipe(
               map((stats) => {
+                const precision = 100;
                 return {
                   totalReward: stats.totalReward,
-                  rewardPerTx: stats.totalReward / stats.totalTx,
-                  feePerTx: stats.totalFee / stats.totalTx,
+                  rewardPerTx: Math.round((stats.totalReward / stats.totalTx) * precision) / precision,
+                  feePerTx: Math.round((stats.totalFee / stats.totalTx) * precision) / precision,
```

Comment:
Shouldn't we use `formatAmount` or the template pipe `number:` for all the roundings?

Reply:
Updated https://github.com/mempool/mempool/pull/1445/commits/c5837ab9df1ba1424375eae3eb170f1d9748b985